src/features/commentApi.jsx

```diff
@@ -0,0 +1,87 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const addComment = createAsyncThunk(
+  "post/comment",
+  async ({   commentData, _id }) => {
+    const token = localStorage.getItem("token");
```

Comment:
instead of accessing token every time from localstorage i would suggest you to keep it in auth store